.env

```diff
@@ -28,6 +28,15 @@ SHANOIR_PREFIX=
 SHANOIR_X_FORWARDED=generate
 
 
+# SHANOIR_CERTIFICATE: Generation of the TLS server certificate
+# 
+# Possible values:
```

Comment:
Does INRIA provide valid certificates ? Otherwise we could propose let's encrypt (via certbot) ?

Reply:
yes, and we are specifically requested *not* to use let's encrypt